src/wymeditor/editor/base.js

```diff
@@ -98,7 +98,6 @@ WYMeditor.editor.prototype._init = function () {
 
     // Construct the iframe
     iframeHtml = wym._options.iframeHtml;
-    iframeHtml = h.replaceAllInStr(iframeHtml, WYMeditor.INDEX, wym._index);
```

Comment:
We need to document this as a backwards-incompatible change, though. I think some plugin authors relied on `INDEX`